README.md

```diff
@@ -1,70 +1,63 @@
-# Getting Started with Create React App
+## Bookstore
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+In this project, I build a Bookstore a website with two subpages: "Books" and "Categories". This MVP version allows to:
 
-## Available Scripts
+- Display a list of books.
+- Add a book.
+- Remove a selected book.
 
-In the project directory, you can run:
+## Built With
 
-### `yarn start`
+- Java Script
+- React
+- Redux
+- HTML
+- CSS
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+### Technologies used
 
-The page will reload if you make edits.\
-You will also see any lint errors in the console.
+- Github
+- GitFlow
+- Visual Studio Code
+- webpack
+- npm
+- Heroku deployment
 
-### `yarn test`
+## Demo
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+![](demo.gif)
+[Netlify](https://keen-lamport-984a6a.netlify.app/app)
+[Heroku](https://calculator-app-shoira.herokuapp.com/)
 
-### `yarn build`
+## Getting Started
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+To get a local copy up and running follow these simple example steps.
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+- go to the directory that you want to clone the repository in.
+- right click and choose Git Bash here.
+- type "git clone https://github.com/ShoiraTa/Calculator.git".
+- run `npm install` to install all the dependencies
```

Comment:
- [OPTIONAL] You want to consider fixing the link for the cloning step in your README. ♻️